package-lock.json

```diff
@@ -493,70 +493,70 @@
       "dev": true,
       "license": "MIT",
       "dependencies": {
         "@autorest/codemodel": "~4.20.0",
         "@autorest/extension-base": "~3.6.0",
         "@azure-tools/codegen": "~2.10.0",
-        "@azure-tools/typespec-autorest": "^0.43.0",
-        "@azure-tools/typespec-azure-core": "^0.43.0",
-        "@azure-tools/typespec-azure-resource-manager": "^0.43.0",
-        "@typespec/compiler": "^0.57.0",
-        "@typespec/http": "^0.57.0",
-        "@typespec/openapi": "^0.57.0",
-        "@typespec/openapi3": "^0.57.0",
-        "@typespec/prettier-plugin-typespec": "^0.57.0",
-        "@typespec/rest": "^0.57.0",
-        "@typespec/versioning": "^0.57.0",
+        "@azure-tools/typespec-autorest": "^0.44.0",
+        "@azure-tools/typespec-azure-core": "^0.44.0",
+        "@azure-tools/typespec-azure-resource-manager": "^0.44.0",
+        "@typespec/compiler": "^0.58.0",
+        "@typespec/http": "^0.58.0",
+        "@typespec/openapi": "^0.58.0",
+        "@typespec/openapi3": "^0.58.0",
+        "@typespec/prettier-plugin-typespec": "^0.58.0",
+        "@typespec/rest": "^0.58.0",
+        "@typespec/versioning": "^0.58.0",
         "change-case-all": "~2.1.0",
         "lodash": "~4.17.20",
         "pluralize": "^8.0.0",
         "prettier": "~3.1.0"
       }
     },
     "node_modules/@autorest/openapi-to-typespec/node_modules/@azure-tools/typespec-autorest": {
-      "version": "0.43.0",
-      "resolved": "https://registry.npmjs.org/@azure-tools/typespec-autorest/-/typespec-autorest-0.43.0.tgz",
-      "integrity": "sha512-tZ9uXXOdxu2y01W9n3mzXf+IEY2MQkP/JaQaKlcOx2+dbklHNQWSDU0Vm6Gmm6l//XiF9QiI8653BiuO97czyw==",
+      "version": "0.44.1",
+      "resolved": "https://registry.npmjs.org/@azure-tools/typespec-autorest/-/typespec-autorest-0.44.1.tgz",
+      "integrity": "sha512-lw/iM659GuFgckDeRFFu0vx6wGBy814n+mjzbpi0Qwjvj8/hYULSjpty9P4WBDE30rYCUde1pWX5nK6TnwhOkQ==",
       "dev": true,
       "license": "MIT",
       "engines": {
         "node": ">=18.0.0"
       },
       "peerDependencies": {
-        "@azure-tools/typespec-azure-core": "~0.43.0",
-        "@azure-tools/typespec-azure-resource-manager": "~0.43.0",
-        "@azure-tools/typespec-client-generator-core": "~0.43.0",
-        "@typespec/compiler": "~0.57.0",
-        "@typespec/http": "~0.57.0",
-        "@typespec/openapi": "~0.57.0",
-        "@typespec/rest": "~0.57.0",
-        "@typespec/versioning": "~0.57.0"
+        "@azure-tools/typespec-azure-core": "~0.44.0",
+        "@azure-tools/typespec-azure-resource-manager": "~0.44.0",
+        "@azure-tools/typespec-client-generator-core": "~0.44.2",
+        "@typespec/compiler": "~0.58.0",
+        "@typespec/http": "~0.58.0",
+        "@typespec/openapi": "~0.58.0",
+        "@typespec/rest": "~0.58.0",
+        "@typespec/versioning": "~0.58.0"
       }
     },
     "node_modules/@autorest/openapi-to-typespec/node_modules/@azure-tools/typespec-azure-core": {
```

Comment:
Looks like this was already a problem, but I just noticed openapi-to-typespec is still not aligned with the version ranges used by other packages, so we're still loading multiple versions side-by-side

Reply:
@pshao25 can you take a look at fixing these dependencies?

Reply:
We are not ready for 0.59. It still needs some time. 

Will update after we are ready.